docs/guide/bundles.md

```diff
@@ -15,11 +15,11 @@ Refer to their respective pages for instructions on how to install them.
 
 ### How to create bundles?
 
-See [Symfony documentation on bundles](https://symfony.com/doc/current/bundles.html) to learn how to structure your bundle.
+See [Symfony documentation on bundles](https://symfony.com/doc/4.3/bundles.html) to learn how to structure your bundle.
 
 ### How to remove a bundle?
 
-To remove a bundle (either one you created yourself, or an out-of-the-box one that you do not need) see the [How to Remove a Bundle](http://symfony.com/doc/current/bundles/remove.html) instruction in Symfony doc.
+To remove a bundle (either one you created yourself, or an out-of-the-box one that you do not need) see the [How to Remove a Bundle](http://symfony.com/doc/3.4/bundles/remove.html) instruction in Symfony doc.
```

Comment:
```suggestion
To remove a bundle (either one you created yourself, or an out-of-the-box one that you do not need) see the [How to Remove a Bundle](http://symfony.com/doc/4.3/bundles/remove.html) instruction in Symfony doc.
```

Reply:
The section does not exist in 4.3 Symfony doc. The link to 3.4 version left intentionally, as indicated in the PR desc. :)